routers/repo/setting.go

```diff
@@ -105,6 +105,14 @@ func SettingsPost(ctx *context.Context, form auth.RepoSettingForm) {
 		}
 
 		if repo.IsMirror {
+			if isNameChanged {
+				var err error
+				ctx.Repo.Mirror, err = models.GetMirror(repo.ID)
+				if err != nil {
+					ctx.Handle(500, "RefreshRepositoryMirror", err)
```

Comment:
Amended to include the return statement. Are you ok with "RefreshRepositoryMirror" as the title for the error handling?